### PR TITLE
Requested by section [KIR-62]

### PR DESCRIPTION
**Pull request by SelcukEdma**
Added RequestedBy Section 
Currently is hard coded by I will update when Auth is done
Screen Shot 2019-12-04 at 2 11 34 PM

### CODE REVIEW

File: src/Components/Scenes/RequestAssets/RequestedBy/RequestedBy-Container.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
+import RequestedBy from './RequestedBy';
+
+// This will get updated when Auth is done
+const mapStateToProps = ({ requestAssets }) => {
+  return {
+    requestedBy: `Scott Fowles`
```
Comment (davidkliewer-kirby) [inline src/Components/Scenes/RequestAssets/RequestedBy/RequestedBy-Container.js]: FYI:  I don't see a human friendly name on the SAML token.  We may need to have an additional call that gets the currentUsers details.

Maybe there could be a new endpoint that that returns user details from an email address.  Perhaps this call could become the userPreferences down the road.

Comment (sfowles) [reply src/Components/Scenes/RequestAssets/RequestedBy/RequestedBy-Container.js]: We could alternatively display email here, if there's no good way to deduce the friendly name. Worth considering that we have lots of folks here with hyphenated first and/or last names

File: src/Components/Scenes/RequestAssets/RequestedBy/RequestedBy-Container.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
+import RequestedBy from './RequestedBy';
+
+// This will get updated when Auth is done
```
Comment (Jake-Lindvall) [inline src/Components/Scenes/RequestAssets/RequestedBy/RequestedBy-Container.js]: Maybe tag it with a `TODO` just in case 👍

**Review by Jake-Lindvall — DISMISSED** [review]
Other than a small nitpick and David/Scott's discussion (I'm good with email), this looks awesome!

File: src/Components/Scenes/RequestAssets/RequestedBy/RequestedBy-Container.js (hunk 1 above)
Comment (SelcukEdma) [reply src/Components/Scenes/RequestAssets/RequestedBy/RequestedBy-Container.js]: @davidkliewer-kirby  It seems to me more of userPreference. I think SAML token provide employee email.
@sfowles  thats good point. I just checked the old Kirby we are using email there. So i will display email.

File: src/Components/Scenes/RequestAssets/RequestedBy/RequestedBy-Container.js (hunk 2 above)
Comment (SelcukEdma) [reply src/Components/Scenes/RequestAssets/RequestedBy/RequestedBy-Container.js]: added  `TODO`

**Review by Jake-Lindvall — APPROVED** [review]
Looks great

**Review by lheld-pnmac — APPROVED** [review]
Looks good to me. I'm assuming that styling/responsiveness is going to be addressed in later work?

**Comment by SelcukEdma** [comment]
> Looks good to me. I'm assuming that styling/responsiveness is going to be addressed in later work?

I know Styling is janky :). But We will need to address it. Scott willing to take these styling tasks

**Pull request merged**